jvmti-access/src/main/java/co/elastic/otel/UniversalProfilingCorrelation.java

```diff
@@ -184,8 +198,7 @@ private static MethodHandle generateVirtualChecker() {
       return MethodHandles.lookup().unreflect(isVirtual);
     } catch (NoSuchMethodException | IllegalAccessException | InvocationTargetException e) {
       // virtual threads are not supported, therefore no thread is virtual
-      return MethodHandles.dropArguments(
-          MethodHandles.constant(boolean.class, false), 0, Thread.class);
+      return null;
```

Comment:
[question] why do we don't need to call `dropArguments` anymore here ? Was it not necessary in the first place ?

Reply:
I changed the constant to be `null` instead of being a `MethodHandle` always returning `false`.
I did this to have an easy check for whether virtual threads are available on the platform or not (comparing the constant against null).